bench-vortex/tpch/q7.sql

```diff
@@ -8,7 +8,7 @@ from
         select
             n1.n_name as supp_nation,
             n2.n_name as cust_nation,
-            extract(year from l_shipdate) as l_year,
+            extract(year from CAST(l_shipdate AS date)) as l_year,
```

Comment:
so, we do this b/c Vortex doesn't support loading Date/Time arrays from Arrow, working on fixing that separately.

This also made obvious some issues in the helper functions upstream

Reply:
this was fixed as part of #480